yt_dlp/extractor/goodgame.py

```diff
@@ -9,16 +9,16 @@
 
 class GoodGameIE(InfoExtractor):
     IE_NAME = 'goodgame:stream'
-    _VALID_URL = r'https?://goodgame\.ru/channel/(?P<id>\w+)'
+    _VALID_URL = r'https?://goodgame\.ru/(?P<id>\w+)'
```

Comment:
As the old URLs still work, let's also match them
```suggestion
    _VALID_URL = r'https?://goodgame\.ru/(?:channel/)?(?P<id>\w+)'
```